src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,26 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieService.class);
```

Comment:
The `injector.getInstance` method is incorrectly used here. You are trying to get an instance of `MovieSessionService` but are passing `MovieService.class` to the injector. This will cause a `ClassCastException`. Ensure you pass the correct class type to the injector.